### PR TITLE
Move propPath predicate functions to path

### PR DESCRIPTION
**Pull request by evilsoft**
### The Last Update
image

This PR addresses the last two items on [this issue][issue] by moving the `propPath` predicate function over to their `path` names:
* `propPathEq` -> `pathEq`
* `propPathSatisfies` -> `pathSatisfies`

Once these are merged we can close out that issue and ship the long awaited `0.12.0`

[issue]: https://github.com/evilsoft/crocks/issues/329

### CODE REVIEW

File: src/predicates/pathSatisfies.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const isArray = require('../core/isArray')
+const isEmpty  = require('../core/isEmpty')
+const isInteger = require('../core/isInteger')
+const isNil = require('../core/isNil')
+const isPredOrFunc = require('../core/isPredOrFunc')
+const isString = require('../core/isString')
+const predOrFunc = require('../core/predOrFunc')
+
+const err = name =>
+  `${name}: First argumnet must be an Array of Non-empty Strings or Integers`
```
Comment (dalefrancis88) [inline src/predicates/pathSatisfies.js]: mis-spelling of argument, this is the start of it and there is a bunch of copy-🍝 below.

File: src/predicates/propPathSatisfies.spec.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ test('propPathSatisfies errors', t => {
     x => propPathSatisfies(x, T, 'target')
   )
 
-  const err = /propPathSatisfies: Array of Non-empty Strings or Integers required for first argument/
+  const err = /propPathSatisfies: First argumnet must be an Array of Non-empty Strings or Integers/
```
Comment (evilsoft) [inline src/predicates/propPathSatisfies.spec.js]: Also will fix these in the specs. Great 👀

**Comment by evilsoft** [comment]
Bummer, need to add back the docs files, just wiped them :smh:

**Comment by dalefrancis88** [comment]
Lol yeah i was wondering where they went :) the danger of force-push

File: src/predicates/propPathSatisfies.spec.js [hunk 3]
```diff
@@ -56,7 +56,7 @@ test('propPathSatisfies errors', t => {
     x => propPathSatisfies([ 'a' ], x, 'target')
   )
 
-  const predErr = /propPathSatisfies: Pred or predicate function required for second argument/
+  const predErr = /propPathSatisfies: Second argument must be a Pred or predicate function/
```
Comment (bennypowers) [inline src/predicates/propPathSatisfies.spec.js]: Nits: would like a small "n" on `Non-empty` and a big "F" on `function`

**Comment by evilsoft** [comment]
Okay @bennypowers and @dalefrancis88 I was able to get all the requested changes added and still include the docs. 🔴 👁 for some final review!

**Pull request merged**